src/services/accessValidation.js

```diff
@@ -66,3 +67,23 @@ export async function validateUserAuthForAdmin(userId) {
     throw error;
   }
 }
+
+export async function getUserReadRegions(userId) {
+  try {
+    const readRegions = await Permission.findAll({
+      attributes: ['regionId'],
+      where: {
+        userId,
+        [Op.or]: [
+          { scopeId: SCOPES.READ_WRITE_REPORTS },
+          { scopeId: SCOPES.READ_REPORTS },
+          { scopeId: SCOPES.APPROVE_REPORTS },
```

Comment:
Do we want users that can approve reports to be able to see all reports for that region?

Reply:
I can see that could go either way. Let's verify with Patrice.

Reply:
Looks like she is ok with these per the slack communication. We can always revisit in the future if anything changes.